config/environments/production.rb

```diff
@@ -40,7 +45,7 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  # config.force_ssl = true
```

Comment:
Is this intentional?  I think we did this on purpose in #13

Reply:
Good catch, thanks.